Cargo.toml

```diff
@@ -33,15 +37,21 @@ hex = "0.4.3"
   version = "1.0.111"
   features = [ "derive", "rc" ]
 
+[patch.crates-io]
+# Use this branch until it is merged into bls_dkg master.
+bls_dkg = { git = "https://github.com/maidsafe/bls_dkg", branch="blsttc_2_3_0_pr" }
```

Comment:
https://github.com/maidsafe/bls_dkg/pull/94 is merged now

Reply:
cool, using published version now.